README.md

```diff
@@ -0,0 +1,36 @@
+# Hack with docker
+
+To quickly start hacking, hm-admin comes with a docker ready development environment.
+
+It leverages three services:
+
+- `hm-admin`: The interesting part in our case. Hm-admin is running in this container and hm-admin sources are shared with the host
+- `api` and `mysql`: Those containers are responsible for the API to be developed and tested against. The API is made from https://api-platform.com/.
+
+
+## Initial steps for setting up your docker development environment
+
+First, you don’t want to recreate your database or redownload all node_modules each time you remove your containers so, we create two data containers:
+
+```
+docker create --name hmadmin_node_modules -v /app/node_modules tianon/true
+docker create --name hmadmin_mysql_data -v /var/lib/mysql mysql:5
```

Comment:
Seems to be a good practice to use the same image as runned container. I got no personal opinion on this